format/view-spec.md

```diff
@@ -160,6 +179,56 @@ Each entry in `version-log` is a struct with the following fields:
 | _required_  | `timestamp-ms` | Timestamp when the view's `current-version-id` was updated (ms from epoch) |
 | _required_  | `version-id`   | ID that `current-version-id` was set to |
 
+#### Table Identifier
+
+The table identifier holds a reference, containing a namespace and a name, of a table or view in the catalog.
+
+| Requirement | Field name     | Description |
+|-------------|----------------|-------------|
+| _optional_  | `catalog`      | A string specifying the name of the catalog. If set to `null`, the catalog is the same as the view's catalog |
+| _required_  | `namespace`    | A list of namespace levels |
+| _required_  | `name`         | A string specifying the name of the table/view |
+
+### Materialized View Metadata stored as part of the Table Metadata
```

Comment:
we need to think about how to organize this more clearly. we need to make it abundantly clear that this is for table metadata. also the section title is too long.

Maybe call this section `Storage table metadata` to match the `View metadata` section name at the same level.